src/SConscript

```diff
@@ -12,7 +12,8 @@ try:
 except:
     # If SCons is using an old python (2.4 is relatively common still), then the above may fail.
```

Comment:
One thing I forgot to say was that the above code *always* fails so I'm not entirely sure what the purpose of the try block is. The comment is obviously out of date.